### PR TITLE
fronius: extend if battery storage is involved

### PR DESCRIPTION
**Pull request by csett86**
## Proposed change
The energy dashboard for fronius needs more Riemann sum integration if battery storage is involved

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @csett86, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21072).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/fronius.markdown [hunk 1]
```diff
@@ -80,8 +80,10 @@ Recommended energy dashboard configuration for meter location in feed in path (`
 
 - For `Grid consumption` use the meters `Energy real consumed` entity.
 - For `Return to grid` use the meters `Energy real produced` entity.
-- For `Solar production` add each inverters `Energy total` entity.
-- `Battery systems` aren't supported directly. Use [Riemann sum](/integrations/integration/) with negative values of `Power battery` entity (from power_flow endpoint found in your `SolarNet` device) for charging and positive values for discharging.
+- For `Solar production`: 
+  - If no battery is involved at an inverter: Add each inverters `Energy total` entity.
+  - If a battery is involved at an inverter: Use [Riemann sum](/integrations/integration/) over `Power photovoltaics` entity (from power_flow endpoint found in your `SolarNet` device)
```
Comment (farmio) [inline source/_integrations/fronius.markdown]: So `Energy Total` sums PV and Battery (-)charge, discharge energy?
Oh dear, this API is a mess...

**Review by farmio — DISMISSED** [review]
Saw your forum post. Makes sense (your thread, not Fronius 🙃)
Thank you very much 👍

File: source/_integrations/fronius.markdown (hunk 1 above)
Comment (csett86) [reply source/_integrations/fronius.markdown]: I also understand it from an API perspective, „Energy total“ on the inverter is whatever is converted from DC to AC. Independent of whether its coming from a solar panel or a battery storage.

**Comment by csett86** [comment]
While we are at it: Do you see a possibility to add the riemann sum integrations necessary for a battery storage for the energy dashboard directly to the fronius integration component? I could implement it if this is conceptually possible from HA perspective.

**Comment by farmio** [comment]
That is out of scope of the integration, I'm afraid. See https://discord.com/channels/330944238910963714/857346526527094814/925442360714858578

File: source/_integrations/fronius.markdown (hunk 1 above)
Comment (farmio) [reply source/_integrations/fronius.markdown]:
```suggestion
- For `Solar production`: 
  - If no battery is connected to an inverter: Add each inverters `Energy total` entity.
  - If a battery is connected to an inverter: Use [Riemann sum](/integrations/integration/) over `Power photovoltaics` entity (from power_flow endpoint found in your `SolarNet` device)
```
I'm not particular fluent in english - is "involved at" the correct term?

Comment (csett86) [reply source/_integrations/fronius.markdown]: Indeed, thank you, updated

**Review by farmio — APPROVED** [review]

**Pull request merged**